examples/2d/move_sprite.rs

```diff
@@ -0,0 +1,45 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(sprite_movement.system())
```

Comment:
on main the `.system()` syntax is no longer needed.
So we should probably also remove it here :)

Reply:
it can be done later, we will need a big pass on all examples anyway